### ISSUE TITLE
Migration, live migration and native ceph support

### ISSUE THREAD

**Issue by virento** [OpenNebula/addon-lxdone#22]
Hello, I wanted to check and see if an estimated timeline is known for release on the following features and if so, when that is:

* Migration
* Live migration
* Native LXD Ceph support

Migration is the biggest feature I'm looking forward to however I thought I would ask about a few I had interest in using.

Great project BTW, this is really the future of cloud computing IMO and I hope to see this merged into the regular OpenNebula release sooner than later.

**Comment by dann1** [comment]
Hello, thanks for the comments. Live migration is not being focused now, however, plain migration is second after snapshots in our tasks. I recently worked on native LXD Ceph support https://github.com/lxc/lxd/issues/3993 https://discuss.linuxcontainers.org/t/ceph-datastore-for-several-lxd-nodes/740. But I concluded after some research https://discuss.linuxcontainers.org/t/same-ceph-storage-with-multiple-lxd-hosts/754/2 and a lot of tinkering that LXD's Ceph module is not cloud ready.
